presto-spi/src/main/resources/com/facebook/presto/spi/type/zone-index.properties

```diff
@@ -2186,13 +2186,13 @@
 2177 CST6CDT
 2178 Cuba
 2179 EET
-2180 EST
+# 2180 EST  # not in java.time
 2181 EST5EDT
 2182 Egypt
 2183 Eire
 2184 GB
 2185 GB-Eire
-2186 HST
+# 2186 HST  # not in java.time
```

Comment:
The bottom of this file contains comments about zones not in one or the other of these systems.  I think we should remove that comment since everything is JDK time now.